src/robot.js

```diff
@@ -49,7 +49,7 @@ module.exports = app => {
       }
 
       let validationResult = isReferenceValid(ref)
-      if (!validationResult.result) {
+      if (!validationResult.result && config.brunchyyyDisable === false) {
```

Comment:
Hm ... what will happen if `config. brunchyyyDisable` is `undefined`?
Would the bot behave as you expected?

Reply:
I was thinking about that but I can't come up with a scenario where this can happen. I'm sure there's a way but I'm drawing a blank.